src/backend/common/datafeeds/parsers/parser_base.py

```diff
@@ -3,22 +3,22 @@
 from typing import Generic, TypeVar
 
 
-TModel = TypeVar("TModel")
+TParsedResponse = TypeVar("TParsedResponse")
 
 
 class ParserInputException(Exception):
     pass
 
 
-class ParserBase(abc.ABC, Generic[TModel]):
+class ParserBase(abc.ABC, Generic[TParsedResponse]):
     """
     Provides a basic structure for parsing pages.
     Parsers are not allowed to return Model objects, only dictionaries.
     """
 
     @classmethod
     @abc.abstractmethod
-    def parse(cls, html: bytes) -> TModel:
+    def parse(cls, html: bytes) -> TParsedResponse:
```

Comment:
Note to Zach in the AM - there are two different types of parsers - JSON parsers and HTML parsers. HTML parsers can take bytes, but JSON parsers will need to take a Dict. Right now the FMS API parsers don't have a common subclass, so we'll need to namespace and differentiate these parsers. Or maybe only add a JSON parser for now.